app/src/components/AddFeedbackForm/index.js

```diff
@@ -0,0 +1,161 @@
+    } = this.props;
+    return (
+      <div>
+        {message &&
+          <Heading
+            tag="h4"
+            align="center"
+          >
+            <Paragraph>
+              {message}
+            </Paragraph>
+          </Heading>
+        }
+      <Headline
+        size="small"
+        strong
+      >
+        We, Thank You For Your Feedback!
+      </Headline>
+      <Heading
+        tag="h4"
+        align="center"
+      >
+        Please Fill Out The Form Below...
+      </Heading>
+      <Form>
+        <FormFields>
+          <FormField
+            label="Name *"
+            htmlFor="feedbackNameInput"
+            help="Please make sure your name is correct or Edit your profile to correct it!"
+            error={nameInput.touched && nameInput.error ? nameInput.error : null}
+          >
+            <input
+              {...nameInput}
+              ref="nameInput"
+              className={styles.blockedInput}
+              id="feedbackNameInput"
+              type="text"
+              name="name"
+              className={styles.blockedInput}
+              disabled
+              required
+            />
+          </FormField>
+          <FormField
+            label="URL *"
+            htmlFor="feedbackUrlnput"
+            help="Fill in the URL you want to give feedback about"
+            error={urlInput.touched && urlInput.error ? urlInput.error : null}
+          >
+            <input
+              {...urlInput}
+              ref="urlInput"
+              className={styles.longUrl}
+              id="urlInput"
+              type="text"
+              name="url"
+              disabled
+              required
+            />
+          </FormField>
+          <FormField
+            label="Description"
+            htmlFor="feedbackDescriptionInput"
+            help="Describe your problem/feedback"
+            error={descriptionInput.touched && descriptionInput.error
+               ? descriptionInput.error : null
+            }
+          >
+            <textarea
+              {...descriptionInput}
+              id="feedbackDescriptionInput"
+              ref="feedbackDescriptionInput"
+              type="text"
+              rows="5"
+              cols="40"
+              required
+            />
+          </FormField>
+        </FormFields>
+        <Footer className={styles.footer}>
+          {user ?
+          <Menu direction="row">
+            <Button
+              className={styles.button}
+              label="Submit"
+              primary
+              onClick={onSubmitFeedback}
+            />
+            <Button
+              label="Clear"
+              onClick={onClear}
+            />
+          </Menu>
+          :
+          <Heading
+            tag="h3"
+          >
+            Please Login to contiue giving us your feedback :)
+          </Heading>
+          }
+        </Footer>
+      </Form>
+      </div>
+    );
+  }
+}
+
+AddFeedbackForm.propTypes = {
```

Comment:
Nice!! One common thing is to accidentally add props that you don't end up using.  You did not do that, which is great.  Overall other than the small indentation stuff, this component looks great!  Nice job.